src/styles/blocks/header.scss

```diff
@@ -0,0 +1,51 @@
+.header {
+  background-image: url(../images/header-background.jpg);
+  background-size: cover;
+  background-position: 90% 50%;
+
+  @include onTablet {
+    background-position: center;
+  }
+
+  @include onDesktop {
+    background-position: center 40%;
+  }
+
+  &__content {
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+
+    box-sizing: border-box;
+    height: 100vh;
+    padding: $headerTopPadding 0 88px;
+
+    @include onTablet {
+      padding: $headerTopPaddingTablet 0 130px;
+    }
+
+    @include onDesktop {
```

Comment:
This 204px added together with the element margin of ```h1``` makes the "Take the Streets" text appear in the middle. 

It should be closer to the bottom according to designs